Automattic/woocommerce-payments
# chore: ECE copy for tokenized cart implementation

Fixes https://github.com/Automattic/woocommerce-payments/issues/9721

#### Changes proposed in this Pull Request









Continuing the groundwork to pave the path for a modification of the ECE to leverage the Store API.

I'm creating a copy of the ECE JS, so it can be modified.
This allows us to modify it in isolation, making safe changes.

#### Testing instructions





- Update your WooPayments Dev Tools
- Enable the "Enable Cart-Token implementation for ECE" feature flag
- Restart your webpack build (`npm run start`)
- As a merchant, ensure you have GooglePay/ApplePay enabled in the WooPayments settings (for product pages, cart, checkout)
- As a customer, go to a product page for a simple product
- You should see the GooglePay button in Chrome
- If you look at the dev tools, the `tokenized-express-checkout` JS should be enqueued, indicating that the new scripts are being used

-------------------


**Post merge**
